packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -761,6 +761,24 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
   @override
   void jumpTo(double value);
 
+  /// Jumps the scrolling position from current value to delta without animation
```

Comment:
Nit: This could be confused with the `jumpTo` method.
```suggestion
  /// Changes the scrolling position based on a pointer signal from current value to delta without animation
```